src/components/IconButton.tsx

```diff
@@ -22,20 +23,15 @@ interface IconButtonProps {
     | "9xl";
 }
 
-const IconButton: React.FC<IconButtonProps> = ({
-  icon,
-  onClick,
-  className,
-  size,
-  disabled,
-}) => {
-  const baseClassName =
-    "cursor-pointer " +
-    (disabled ? `opacity-40` : "") +
-    (className ? ` ${className}` : "");
-  const computedClassName = size
-    ? `text-${size} ${baseClassName}`
-    : `${baseClassName}`;
+const IconButton: React.FC<IconButtonProps> = ({ icon, onClick, className, size, disabled }) => {
+  const baseClassName = React.useMemo(
+    () => "cursor-pointer " + (disabled ? `opacity-40` : "") + (className ? ` ${className}` : ""),
```

Comment:
I think you can also just put the whole thing in a string i.e.
```js
    () => `cursor-pointer ${disabled ? "opacity-40" : ""} ${className ? className : ""}`,
```
also since className is a string (I hope), why do you need the second ternary?

Reply:
className can be undefined so it'd come in as "cursor-pointer opacity-40 undefined", which is actually fine but feels weird.